### PR TITLE
PEP 755: Status quo of the CPython 3.12 build system

### PR DESCRIPTION
**Pull request by erlend-aasland**
----
:books: Documentation preview :books:: https://pep-previews--3.org.readthedocs.build/

### CODE REVIEW

**Comment by erlend-aasland** [comment]
cc. @zware

File: peps/pep-0755.rst [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+If we are to make large scale changes to the current build system,
+we need to know more about its pros and cons.
+
+This PEP intends to document the current build system;
+how is it perceived by contributors and core developers,
+and what kind of build system they envision in the future.
+
+This PEP can be used as input to decide whether to improve the existing build system,
+rewrite the existing build system,
+or implement a completely new build system.
```
Comment (erlend-aasland) [inline peps/pep-0755.rst:37]: Perhaps this part could be placed under a _Motivation_ header.

File: peps/pep-0755.rst [hunk 2]
```diff
@@ -0,0 +1,193 @@
+PEP: 755
+Title: Status quo of the build system as of CPython 3.12
```
Comment (erlend-aasland) [inline peps/pep-0755.rst:2]: I don't like the title. We need something better.

File: peps/pep-0755.rst (hunk 1 above)
Comment (erlend-aasland) [reply peps/pep-0755.rst:37]: Renamed _Rationale_ as _Motivation_. Feels better.

**Comment by erlend-aasland** [comment]
I'm wondering if this is too small for a PEP. It's hard to say. With the larger project in mind (actually replacing the whole build system), it makes sense to carefully progress PEP by PEP. Replacing the whole build system is a topic bound to generate heated discussions, so I _think_ it makes sense to be overly bureaucratic, and manifest the summarisation of our polls in a (small) PEP.

**Comment by zware** [comment]
I agree with that assessment.  I'm not sure that this is really a PEP per se, but in the same way that I don't think PEPs 482 and 483 are really PEPs.  But it's the mechanism we've got, so a PEP it is :)

**Comment by erlend-aasland** [comment]
> I agree with that assessment. I'm not sure that this is really a PEP per se, but in the same way that I don't think PEPs 482 and 483 are really PEPs. But it's the mechanism we've got, so a PEP it is :)

Exactly :) So, I guess the only remaining issue now is to polish the title and perhaps also adjust wording in the abstract.

**Comment by erlend-aasland** [comment]
Should we post a heads-up on Discourse, or just submit the PEP?

**Comment by zware** [comment]
I think we might want to have the next PEP ready to go before posting this one to help this better fit as a PEP.  I'm still trying to find time to actually read all the way through this one though :(

**Comment by erlend-aasland** [comment]
> I think we might want to have the next PEP ready to go before posting this one to help this better fit as a PEP. I'm still trying to find time to actually read all the way through this one though :(

I agree, and I've thought about the same.

**Pull request closed without merging**